tests/model/liveselection.js

```diff
@@ -872,4 +872,37 @@ describe( 'LiveSelection', () => {
 			expect( values ).to.deep.equal( [] );
 		} );
 	} );
+
+	describe( '_updateAttributes()', () => {
```

Comment:
I've just added two new tests. But I don't have an idea where these tests should be located, so I left it as it is (in new describe).